org.ultimatepp.TheIDE.yml

```diff
@@ -0,0 +1,115 @@
+app-id: org.ultimatepp.TheIDE
+runtime: org.gnome.Platform
```

Comment:
why do you need the gnome platform?

Reply:
TheIDE is using libnotify for notification. Without using org.gnome.Platform there is a compilation error.

Do you know better alternative? Maybe there is a way to add libnotify when your runtime is org.freedesktop.Platform? On the other hand on Flatpak documentation [page](https://docs.flatpak.org/en/latest/available-runtimes.html) dedicated to run times there is an information whenever you need libnotify GNOME platform should be used.

Reply:
libnotify will be in 24.08 of Freedesktop runtime, for now build it. It's small and doesn't have many dependencies.

Reply:
libnotify will be in freedesktop 24.08 runtime. If you need that only, you can build libnotify separately and remove it when bumping to 24.08

Reply:
Hello @bbhtt, Thanks for suggestion! I migrated to org.freedesktop.Platform and I added separate step to build libnotify from scratch.